src/components/Estimate/Estimate.tsx

```diff
@@ -1,63 +1,88 @@
-import React, { ComponentProps, ReactNode, useCallback, useEffect, useMemo, useState } from 'react';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import styled from 'styled-components';
+import { gapXs, gapS, warn0 } from '@taskany/colors';
+import { nullable, Text } from '@taskany/bricks';
+import { IconExclamationSmallOutline } from '@taskany/icons';
 
-import { createDateRange, isPastDate, getQuarterFromDate, getYearFromDate } from '../../utils/dateTime';
+import { createDateRange, isPastDate, getQuarterFromDate, getYearFromDate, getDateString } from '../../utils/dateTime';
 import { DateType, DateRange } from '../../types/date';
-import { EstimateDate } from '../EstimateDate';
+import { EstimateDate } from '../EstimateDate/EstimateDate';
 import { EstimateQuarter } from '../EstimateQuarter';
 import { EstimateYear } from '../EstimateYear';
-import { Option } from '../../types/estimate';
-import { EstimatePopup } from '../EstimatePopup';
 
 import { tr } from './Estimate.i18n';
 
-type EstimateValue = {
+const StyledWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: ${gapS};
+`;
+
+const StyledWarningWrapper = styled.div`
+    display: flex;
+    align-items: center;
+    gap: ${gapXs};
+`;
+
+const StyledWarningCircle = styled.div`
+    width: 12px;
```

Comment:
Sorry, but it's impossible to use custom sizes for icons... U must use @taskany/icons instead. Thanks for understanding.